cmd/minishift/cmd/hostfolder/mount.go

```diff
@@ -67,6 +68,6 @@ var hostfolderMountCmd = &cobra.Command{
 }
 
 func init() {
-	HostfolderCmd.AddCommand(hostfolderMountCmd)
-	hostfolderMountCmd.Flags().BoolVarP(&mountAll, "all", "a", false, "Mounts all defined host folders to the running OpenShift cluster.")
+	HostFolderCmd.AddCommand(mountCmd)
+	mountCmd.Flags().BoolVarP(&mountAll, "all", "a", false, "Mounts all defined host folders into the Minishift VM.")
```

Comment:
Why VM?

Reply:
Because hostfolders is about the VM and its file system, not the OpenShift cluster at all.

Reply:
Also my thinking, so rather see this being consistent

Reply:
It should be now